ruby/google-protobuf.gemspec

```diff
@@ -10,16 +10,17 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.files       = Dir.glob('lib/**/*.rb')
   if RUBY_PLATFORM == "java"
+    s.platform  = "java"
     s.files     += ["lib/google/protobuf_java.jar"]
   else
     s.files     += Dir.glob('ext/**/*')
     s.extensions= ["ext/google/protobuf_c/extconf.rb"]
-    s.add_development_dependency "rake-compiler-dock"
+    s.add_development_dependency "rake-compiler-dock", "~> 0.5.1"
```

Comment:
It was added in response to commentary above about the removal of `Gemfile.lock`. The tests pass with the latest versions of all the gems that have version requirements specified. I believe the concern is that to have completely unpinned development dependencies could introduce flakiness in the automated tests.